libkbfs/md_ops.go

```diff
@@ -295,9 +295,7 @@ func (md *MDOpsStandard) processMetadata(ctx context.Context,
 	*rmds = RootMetadataSigned{}
 	irmd := MakeImmutableRootMetadata(rmd, key, mdID, localTimestamp, true)
 
-	// Revisions created locally should always override anything else
-	// in the cache.
-	err = md.config.MDCache().Replace(irmd, irmd.BID())
+	err = md.config.MDCache().Put(irmd)
```

Comment:
This was just something unrelated I noticed.  Fetched MDs should never replace local MDs in the cache.

Reply:
Could you explain why it should never replace? Is it not only for stuff from server, but also local new MDs?

Reply:
`processMetadata` is only used on the get path, not the put path, so it doesn't get called for new local MDs.  The put path uses Replace, as intended.

Reply:
I see; so using `Replace` here would cause overriding MDs when there are new local MDs not flushed to server. Does it sound correct?